docs/samples/browser-plugin-meetings/app.js

```diff
@@ -2855,20 +2868,17 @@ function removeMember(removeButton) {
 
 function muteMember(muteButton) {
   const meeting = getCurrentMeeting();
-  const unmute = muteButton.getAttribute('data-unmute');
-  const mute = unmute !== 'true';
-
   const participantID = getRadioValue('participant-select');
+  const selectedMember = meeting.members.membersCollection.get(participantID);
+  const mute = selectedMember.isAudioMuted ? 'false' : 'true';
```

Comment:
`const mute` -> `const isMuted`

Reply:
Variable name in the .mute function is expected as mute hence kept it as it is

Reply:
Okay, but `mute` function with the variable name `mute` as an argument may confuse

Reply:
Changes the argument to muteStatus.

Reply:
mute function inside plugin meetings expects the parameter name as mute only else it starts to fail hence kept it like that.